public/index.html

```diff
@@ -30,7 +30,7 @@
     <noscript>
       You need to enable JavaScript to run this app.
     </noscript>
-    <div id="root">
+    <div id="root" class="dp-container">
```

Comment:
it's not more clear `app-container` ? or the same

Reply:
or `webapp-container`

Reply:
The nomenclature of classes that we have been using is `dp` -` component name`, `dp` comes from` doppler`

Reply:
I understand your concern @fcoronelmakingsense.

I think that you are right in that it is not something _related to our brand_. I saw this class as a utility one, something not related to our brand that could be used by a library client to make things easy.

In that case, what do you think about using a name related to the functionality or behavior related to this element?

The question is: what makes this _container_ different than another container, for example, a simple _div_?

Reply:
i talk with @GusBaamonde and we think can use something like `dp-app-container` to mantein the prefix